biogrid/libs/biogrid-simulator/src/lib/biobrain/biobrain-class.ts

```diff
@@ -0,0 +1,24 @@
+import { Brain } from "@biogrid/grid-simulator";
+
+// We can only have one BioBrain per grid
+export class BioBrain implements Brain{
+  private static instance: BioBrain;
+  private states: unknown[] = [];
+  private constructor() { }
+
+  static get Instance(): BioBrain {
+    if (!this.instance) {
+      this.instance = new BioBrain();
+    }
+    return this.instance
+  }
+
+  private set gridStates(states: unknown[]) {
+    this.states = states;
+  }
+
+  sendAction(states: unknown[]): unknown {
```

Comment:
where it says unkown, put a TODO above so that we can easily go back in later and implement